src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,12 @@
-export const UserInfo = () => {};
+import React from 'react';
+
+interface User {
+  name: string,
+  email: string,
+}
```

Comment:
```suggestion
interface Props {
  user: User;
}
```
it's better to accept `user` as single prop, not to destruct